### ISSUE TITLE
Automatic notification about config.json changes through CI

### ISSUE THREAD

**Issue by 6r1d** [hyperledger/iroha#4278]
As we discussed with Vasily, it'll be much more convenient if he's notified about any possible configuration change.
Ultimately, we concluded that it's best to have a CI utility that doesn't use any outside software and notifies him about the changes.
It's not a priority and probably will be added in March 2024.

**Comment by s8sato** [comment]
One of possible solutions
https://github.com/hyperledger/iroha/pull/4276

**Comment by s8sato** [comment]
> [#4309 (comment)](https://github.com/hyperledger/iroha/pull/4309#issuecomment-1957724414)
> 
> This is a new feature introduced by #4276 to automatically label and notify someone of specific code changes. Currently `config-changes` reactions to any change under `configs/`, which may be an inappropriate pattern. I guess `configs/*.template.*` would be correct one at the current directory structure. Since things may have changed after #4239 I'd hear your opinion @0x009922

**Comment by BAStos525** [comment]
> > [#4309 (comment)](https://github.com/hyperledger/iroha/pull/4309#issuecomment-1957724414)
> > This is a new feature introduced by #4276 to automatically label and notify someone of specific code changes. Currently `config-changes` reactions to any change under `configs/`, which may be an inappropriate pattern. I guess `configs/*.template.*` would be correct one at the current directory structure. Since things may have changed after #4239 I'd hear your opinion @0x009922

Yeah, lets's update notifications, please. There should not be a false-positive notifications for DevOps.

**Comment by s8sato** [comment]
Closed as the rest of the issue will be left to #4288
